crypto/pqdsa/internal.h

```diff
@@ -15,22 +15,33 @@ typedef struct {
   int (*pqdsa_keygen)(uint8_t *public_key,
                       uint8_t *private_key);
 
+  int (*pqdsa_sign_message)(const uint8_t *private_key,
+                            uint8_t *sig,
+                            size_t *sig_len,
+                            const uint8_t *message,
+                            size_t message_len,
+                            const uint8_t *ctx_string,
+                            size_t ctx_string_len);
+
   int (*pqdsa_sign)(const uint8_t *private_key,
-                    uint8_t *sig,
-                    size_t *sig_len,
-                    const uint8_t *message,
-                    size_t message_len,
-                    const uint8_t *ctx_string,
-                    size_t ctx_string_len);
+                          uint8_t *sig,
+                          size_t *sig_len,
+                          const uint8_t *message,
```

Comment:
Would you like to refer to it as `digest` or `mu` here and all the way down until the internal/generic functions are used?

Reply:
I think I got them all the way down. I couldn't change the one inside `pkey_pqdsa_verify_generic` as it handles both `message` and `digest` options